doc/release-notes/6895-payara-release-note

```diff
@@ -0,0 +1,5 @@
+##Payara 5
+
+A major upgrade of the application server will provide security updates, access to new features like MicroProfile Config API, and will enable upgrades to other core technology, such as Jakarta EE 8.
```

Comment:
We've already switched to Jakarta EE 8 in pull request #6818.

Reply:
@pdurbin Nice, copied this from a previous issue description. I'll make it more general.

Reply:
updated in https://github.com/IQSS/dataverse/pull/6944/commits/7fa2c71ee37ca5ce9dffd4b640858a134f65e53d